gce/scripts/startup-script.sh

```diff
@@ -19,26 +19,31 @@ set -v
 
 # Talk to the metadata server to get the project id
 PROJECTID=$(curl -s "http://metadata.google.internal/computeMetadata/v1/project/project-id" -H "Metadata-Flavor: Google")
-BUCKET=$(curl -s "http://metadata.google.internal/computeMetadata/v1/instance/attributes/BUCKET" -H "Metadata-Flavor: Google")
 
-echo "Project ID: ${PROJECTID}  Bucket: ${BUCKET}"
-
-# get our file(s)
-gsutil cp "gs://${BUCKET}/gce/"** .
+echo "Project ID: ${PROJECTID}"
 
 # Install dependencies from apt
-apt-get update
-apt-get install -yq openjdk-11-jre-headless
+apt-get install -yq openjdk-8-jdk git
+java -version
+
+# Install updated version of Maven
+wget https://www-us.apache.org/dist/maven/maven-3/3.6.2/binaries/apache-maven-3.6.2-bin.tar.gz -P /tmp
+tar xf /tmp/apache-maven-*.tar.gz -C /opt
+ln -s /opt/apache-maven-3.6.2 /opt/maven
+export M2_HOME=/opt/maven
+export MAVEN_HOME=/opt/maven
+export PATH=${M2_HOME}/bin:${PATH}
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+export MAVEN_OPTS="-Dmaven.wagon.http.ssl.insecure=true -Dmaven.wagon.http.ssl.allowall=true -Dmaven.wagon.http.ssl.ignore.validity.dates=true"
```

Comment:
Not quite sure I understand this part. Is there a reason we want to avoid ssl?

Reply:
It doesn't work in GCE without this.

```
[INFO] Scanning for projects...
Downloading from central: https://repo.maven.apache.org/maven2/com/google/cloud/samples/shared-configuration/1.0.10/shared-configuration-1.0.10.pom
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for com.google.gce.demos:getting-started-gce:1.0-SNAPSHOT: Could not transfer artifact com.google.cloud.samples:shared-configuration:pom:1.0.10 from/to central (https://repo.maven.apache.org/maven2): Unexpected error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty and 'parent.relativePath' points at no local POM @ line 26, column 11
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project com.google.gce.demos:getting-started-gce:1.0-SNAPSHOT (/getting-started-java/gce/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for com.google.gce.demos:getting-started-gce:1.0-SNAPSHOT: Could not transfer artifact com.google.cloud.samples:shared-configuration:pom:1.0.10 from/to central (https://repo.maven.apache.org/maven2): Unexpected error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty and 'parent.relativePath' points at no local POM @ line 26, column 11 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
```